biomage/stage/stage.py

```diff
@@ -130,7 +128,7 @@ def get_branch_ref(chart, token, repo_to_ref=None, return_sha=False):
     # A `git` reference can be git@github.com:hms-dbmi-cellenics/releases
```

Comment:
Can you please update this comment to show the new `https://...` format?

Reply:
what is the format now? https://hms-dbmi-cellenics/releases ?

Reply:
https://github.com/hms-dbmi-cellenics/releases *?